scripts/dev/apply_formatting.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
-find src tst -name "*.hpp" -o -name "*.h" -o -name "*.hh" -o -name "*.cc" -o -name "*.cpp" -o -name "*.c" | xargs -n 1 -I '{}' clang-format -i "{}"
+find src \( -name "*.hpp" -o -name "*.h" -o -name "*.hh" -o -name "*.cc" -o -name "*.cpp" -o -name "*.c" \) | xargs -n 1 -P 10 -I '{}' clang-format -style=file -i "{}"
+find tst \( -name "*.hpp" -o -name "*.h" -o -name "*.hh" -o -name "*.cc" -o -name "*.cpp" -o -name "*.c" \) | xargs -n 1 -P 10 -I '{}' clang-format -style=file -i "{}"
```

Comment:
Nevermind this for now, I need to tweak the command anyway.  Nothing related to this work.